python/lsst/obs/base/ingest_tests.py

```diff
@@ -76,38 +79,31 @@ class IngestTestBase(metaclass=abc.ABCMeta):
     observations).
     """
 
+    instrument = ""
+    """The fully qualified name of the instrument.
+    """
+
+    instrumentName = ""
```

Comment:
Same as above, don't ask people to tell you things you can work out for yourself from the fully qualified name of the instrument.